plexpy/notification_handler.py

```diff
@@ -293,24 +310,26 @@ def build_notify_text(session, state):
         full_title = session['title']
 
     # Generate a combined transcode decision value
-    if session['video_decision']:
+    transcode_decision = ''
+    if session.get('video_decision', None):
```

Comment:
I'll change it to `if 'video_decision' in session:`.

I need some way to differentiate between session notifications and timeline notifications. I'll clean up `notification_handler`.